src/crewai/crew.py

```diff
@@ -280,11 +286,16 @@ def kickoff(
             )
 
         metrics = metrics + [
-            agent._token_process.get_summary() for agent in self.agents
+            agent._token_process.get_summary()
+            for agent in self.agents
+            if not isinstance(agent, CustomAgent)
```

Comment:
Ditto. Ideally, all these checks would not exist and we could provide (one by one) wrappers for the famous agents, which would also allow others to implement agents that would respect the interface and respond to all required methods, some times doing nothing.